app/main.py

```diff
@@ -1,3 +1,68 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
Change your type annotation to avoid using Union and placing class in quotes
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```
In case you face errors, use `from __future__ import annotations` import

Reply:
Done